.woodpecker/docker.yml

```diff
@@ -3,10 +3,10 @@ depends_on:
   - web
 
 variables:
-  - &golang_image 'golang:1.18'
+  - &golang_image 'golang:1.19.3'
```

Comment:
Why not `1.19`?

Reply:
more pinning ... if it goes wrong as with 1.18 again ...

Reply:
... well it's an experiment for now if it works or if I need to dig into xgo :(